common/build/eslint-config-fluid/.prettierignore

```diff
@@ -1,4 +1,4 @@
-# Generated by npm / Lerna
+# Generated by npm
 package-lock.json
```

Comment:
we stopped using this file when we moved off lerna to pnpm. I think we can just remove `package-lock.json` from this entirely.

That said, do we even use this config at all anymore since its for prettier? Maybe leave it for now (this pr), but perhaps we can remove this file soon?

Reply:
Good question RE: prettier. I'm going to leave this as-is in this PR and follow up where prettier is still used (server for sure, not sure where else) and should be cleaned up.